### PR TITLE
PMM-7873: backup restore list

### PR DESCRIPTION
**Pull request by fabio-silva**
https://github.com/Percona-Lab/pmm-submodules/pull/1715

What this PR does / why we need it: adds the "Restore history" view to Backup Management

Which issue(s) this PR fixes: https://jira.percona.com/browse/PMM-7873

### CODE REVIEW

File: public/app/percona/backup/Backup.messages.ts [hunk 1]
```diff
@@ -3,6 +3,72 @@ export const Messages = {
...
+        logical: 'Logical',
+      },
+      actions: 'Actions',
+    },
+  },
+  restoreHistory: {
+    table: {
+      noData: 'No Restores found',
+      columns: {
+        started: 'Started At',
```
Comment (tiagomotasantos) [inline public/app/percona/backup/Backup.messages.ts:41]: `Started At`, `No Restores found` and `In progress` are not consistent with what words start with capital letter and what not. I would keep everything lower case except the first word in this case.

File: public/app/percona/backup/Backup.messages.ts [hunk 2]
```diff
@@ -3,6 +3,72 @@ export const Messages = {
...
+      noData: 'No storage locations found',
+      columns: {
+        name: 'Name',
+        type: 'Type',
+        path: 'Endpoint or path',
+        labels: 'Labels',
+        actions: 'Actions',
+      },
+    },
+    addSuccess: 'Backup location was successfully added',
```
Comment (tiagomotasantos) [inline public/app/percona/backup/Backup.messages.ts:56]: Like this 👌

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/percona/backup/Backup.messages.ts (hunk 1 above)
Comment (fabio-silva) [reply public/app/percona/backup/Backup.messages.ts]: good point 👍

File: public/app/percona/backup/Backup.messages.ts (hunk 2 above)
Comment (fabio-silva) [reply public/app/percona/backup/Backup.messages.ts:56]: 👍

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**